app/web/features/auth/jail/ActivenessProbeSection.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  FormControl,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Typography,
+} from "@mui/material";
+import Button from "components/Button";
+import { useTranslation } from "i18n";
+import { AUTH, GLOBAL } from "i18n/namespaces";
+import { ActivenessProbeResponse } from "proto/jail_pb";
+import React, { useState } from "react";
+import { service } from "service";
+
+type StillHosting = "yes" | "no";
```

Comment:
Why don't we make this a boolean?

Reply:
Because I tried and I did not know what I was doing :face_in_clouds: